spec/beaker-pe/install/pe_utils_spec.rb

```diff
@@ -860,10 +860,11 @@ def slice_installer_options(host)
 
     it 'sets puppet-agent acceptable_exit_codes correctly for config helper on upgrade' do
       hosts = make_hosts({
-                           :previous_pe_ver => '3.0',
-                           :pe_ver => '4.0',
-                           :roles => ['agent'],
-                         }, 2)
+        :previous_pe_ver => '3.0',
```

Comment:
@kevpl :previous_pe_ver is retaining the original version we install.  This is otherwise lost (without hacking back into the beaker yaml), once pe_ver is replaced by pe_upgrade_ver.